src/server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const pathname = normalizedURL.pathname;
+
+  if (!pathname.startsWith('/file/')) {
+    res.writeHead(400, 'Bad request');
+    res.end('Please enter valid file path, e.g. "/file/index.html"');
+  }
+
+  const pathSegments = normalizedURL.pathname.split('/file/');
+  const fileName = pathSegments[pathSegments.length - 1];
+
+  const folderPath = './../public/';
```

Comment:
When the path starts from `../` there is no need to add `./`
```suggestion
  const folderPath = '../public/';
```